.circleci/ci/accessibility-axe.js

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+const fs = require('fs');
+const ci = new (require('./ci.js'))();
+const AxeBuilder = require('@axe-core/webdriverjs');
+const WebDriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const { createHtmlReport } = require('axe-html-reporter');
+
+
+const calculateAccessibility = async () => {
+
+    const options = new chrome.Options();
+    const driver = new WebDriver.Builder().forBrowser('chrome').setChromeOptions(options).build();
+
+    try {
+        await driver.get(process.env.ACCESSIBILITY_COLLATERAL_URL);
+
+        await driver.wait(async () => {
+          const readyState = await driver.executeScript('return document.readyState');
+          return readyState === 'complete';
+        });
+//        await driver.findElement(WebDriver.By.xpath("//button[@aria-expanded='false']/coral-accordion-item-label[contains(text(),'Sign in locally')]")).click();
+        const usernameInput = await driver.findElement(WebDriver.By.id('username'));
+        await usernameInput.sendKeys(process.env.LOCAL_USERNAME);
```

Comment:
This is aem specific ? Why is this added in environment variables. You can check a sample here, https://github.com/adobe/aem-core-forms-components/blob/master/.circleci/ci/it-tests.js#L30

Reply:
Please delete these env variables

Reply:
done, use ui.tests pom.xml to find username and password